src/perf/dbserver/QuicPerformanceDataServer/Data/PerformanceContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore;
+using QuicDataServer.Models.Db;
+
+namespace QuicDataServer.Data
+{
+    public class PerformanceContext : DbContext
+    {
+#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
```

Comment:
These warning disable/enable are a bit annoying throughout. Possibly just better to ignore it in the project settings?

Reply:
I don't want to ignore it completely in the project settings. There are a lot of cases where we do want this warning. Its just an inefficiency with data models created with reflection and NRTs.

Reply:
Hmmm... It makes the code quite ugly in my opinion. What if we just disable it for the whole file (at the top) for these data model files? Another thought, what if we put all the data model classes in a single file with the ignore?

Reply:
Standard is 1 file per class, so I don't want to combine them into a single file. But I can just put the disable at the top of each one.

Reply:
From my experience with C# it hasn't been that uncommon to have multiple related classes in a single file. Generally, I agree one per file, but for things like this, combining seem better to me.

Reply:
Actually, never pushed that combine, but got a suggestion for a better way to do this without disabling the warning. So done.